elks/include/linuxmt/homebrew.h

```diff
@@ -0,0 +1,11 @@
+#ifndef __LINUXMT_HOMEBREW_H
+#define __LINUXMT_HOMEBREW_H
+
+/* APIs for homebrew computers */
+
+#include <linuxmt/types.h>
+#include <stddef.h>
+
+byte_t int88 (byte_t fn);
```

Comment:
May I suggest using `int` rather than an 8-bit value for both the argument and return value for `int88`? Allowing a 16-bit return value allows for returning any value including pointers, and a benefit of not passing a char allows the C compiler to zero-extend onto the stack, alleviating the need for extra instructions in the .S file for accessing the argument.

In general, both linuxmt/types.h and stddef.h aren't needed; for instance if just `byte_t` were needed, including arch/types.h would be all that's needed.